ntfysh_client/MainForm.cs

```diff
@@ -69,8 +72,22 @@ private void OnNotificationReceive(object sender, NotificationReceiveEventArgs e
             };
 
             string finalTitle = string.IsNullOrWhiteSpace(e.Title) ? $"{e.Sender.TopicId}@{e.Sender.ServerUrl}" : e.Title;
-            
-            notifyIcon.ShowBalloonTip((int)TimeSpan.FromSeconds((double)Program.Settings.Timeout).TotalMilliseconds, finalTitle, e.Message, priorityIcon);
+
+            if (Program.Settings.UseNativeWindowsNotifications)
+            {
+                notifyIcon.ShowBalloonTip((int)TimeSpan.FromSeconds((double)Program.Settings.Timeout).TotalMilliseconds, finalTitle, e.Message, priorityIcon);
+            }
+            else
+            {
+                this.notificationDialog.ShowNotification(
+                    title: finalTitle, 
+                    message: e.Message, 
+                    timeout_ms: (int)TimeSpan.FromSeconds((double)Program.Settings.Timeout).TotalMilliseconds, 
```

Comment:
This seems a little cursed. If that's the best way to do it, fair enough, but if there's a better way (such as changing the type of `timeout_ms`) that might be better because we're throwing away precision.

Reply:
I'll change it to seconds and drop the whole conversion. Only reason I made it take mS originally was to create a minimal diff with the old method.

Reply:
interesting....

Settings.Timeout is a `decimal`, but the UI element for changing it is set to `increment=1`. Which means that the only way to get it to not be an int would be to edit the json file by hand...